### PR TITLE
Very small note to address #195

### PR DESCRIPTION
**Pull request by allaway**
I don't believe that this needs more discussion than this. It's nice to mention, but a digression to discuss at length.

### Purpose

#195 

### Directions for reviewers

What do you think? Anything more needed? 

#### Which areas should receive a particularly close look?

The one sentence I added. :) 

#### Is the pull request ready for review?

yes
### Manuscript checklist

Unless otherwise noted above, this PR will be considered ready for review when all four items have been checked.

- [ ] All changes to text follow "one sentence per line" [[Manubot instructions](https://github.com/jaybee84/ml-in-rd/blob/draft-branch/USAGE.md#manubot-markdown)]
- [ ] All citations follow the [Manubot citation instructions](https://github.com/jaybee84/ml-in-rd/blob/draft-branch/USAGE.md#citations)
- [ ] All changes or additions to tables follow the [Manubot table instructions](https://github.com/jaybee84/ml-in-rd/blob/draft-branch/USAGE.md#tables)
- [ ] All figures follow the [Manubot figure instructions](https://github.com/jaybee84/ml-in-rd/blob/draft-branch/USAGE.md#figures)
- [ ] All terms that need to be added to glossary have been added to #50 
- [ ] There are no new spelling errors identified by the Manubot spellchecker

### CODE REVIEW

File: content/09.outlook.md [hunk 1]
```diff
@@ -5,6 +5,7 @@ Small sample size, while significant, is not the only roadblock towards applicat
 The high dimensionality of modern data requires creative approaches, such as learning new representations of the data, to manage the curse of dimensionality.
 Leveraging prior knowledge and transfer learning methods to appropriately interpret data is also required.
 Furthermore, we posit that researchers applying machine learning methods on rare disease data should use techniques that increase confidence (i.e., bootstrapping) and penalize complexity of the resultant models (i.e., regularization) to enhance the generalizability of their work. 
+Beyond rare disease, these approaches may also be useful in the context of other areas where a paucity of data makes analysis difficult, such as precision medicine applications in which customized treatment plans are developed for an individual patient's unique genotype and phenotype.
```
Comment (allaway) [inline content/09.outlook.md]:
```suggestion
Beyond rare disease, the approaches described in the previous sections may also be useful in the context of other areas where a paucity of data makes analysis difficult, such as precision medicine applications in which customized treatment plans are developed for an individual patient's unique genotype and phenotype. 
```

File: content/09.outlook.md [hunk 2]
```diff
@@ -5,6 +5,7 @@ Small sample size, while significant, is not the only roadblock towards applicat
 The high dimensionality of modern data requires creative approaches, such as learning new representations of the data, to manage the curse of dimensionality.
 Leveraging prior knowledge and transfer learning methods to appropriately interpret data is also required.
 Furthermore, we posit that researchers applying machine learning methods on rare disease data should use techniques that increase confidence (i.e., bootstrapping) and penalize complexity of the resultant models (i.e., regularization) to enhance the generalizability of their work. 
+Beyond rare disease, the approaches described in the previous sections may also be useful in the context of other areas where a paucity of data makes analysis difficult, such as precision medicine applications in which customized treatment plans are developed for an individual patient's unique genotype and phenotype.
```
Comment (jaclyn-taroni) [inline content/09.outlook.md:8]: It might be a bit long, but I don't have a better/constructive suggestion!

**Review by jaclyn-taroni — APPROVED** [review]
👍🏻  my only comment is that the sentence seems on the long side, but I'm not sure how I'd split it up.

**Comment by allaway** [comment]
Thanks for the review @jaclyn-taroni !

**Pull request merged**